### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by denominator1** [mangecoeur/Citer#14]
Perhaps I miss something essential but I can't get Citer properly run with Sublime Text 3. My package settings file for Citer on my Mac is as follows:

{
    //REQUIRED:

    "bibtex_file_path":  "/Users/myname/ownCloud/test/My_Papers.bib",

    //OPTIONAL:

    //By default Citer Search looks for your keyword in the 
    //author, title, year, and Citekey (id) fields
    "search_fields": ["author", "title", "year", "id"] ,
    //Default format is @Citekey
    "citation_format": "@%s",
    //list of scopes. Could be top level "text" or "source", or limit to
    // e.g "text.html.markdown"
    "completions_scopes": ["text"],
    "enable_completions": true
}

After firing up the search field and typing in an author's name or paper title nothing happens. Citer acts as if it can't find any references although the library does contain antries. Perhaps the main issue is that the file "Citer.sublime-settings" in the package folder "Citer" does not contain any entries (0 Byte)? I removed "citer" and re-installed it, but without any success. Help is much appreciated!

**Comment by michaelstepner** [comment]
I'm having the same issue: Citer does absolutely nothing on my system.  Like @denominator1, I've set up the config file to point to my bibtex file.

But when I run **Citer: Show All** from the command palette, nothing happens. When I run **Citer: Search**, the search bar shows up, but when I search for something and hit enter nothing happens.

**Comment by mangecoeur** [comment]
can you open the sublime console and copy-paste anything that might be there (when you start ST it should have a bunch of stuff about loading packages etc - especially see if there are any errors reported). Hopefully there is something there since otherwise i don't really know where to start looking for your error...

Also, what are you using to generate the bibtex files (I have only tested ones generated by Mendeley).

**Comment by michaelstepner** [comment]
Gah, I realize this is the least useful answer... but it's now working.  I opened the console, saw nothing unusual, tried Citer again, and it worked.

To be clear, when it wasn't working for me, I tried:

* Closing ST3 and reopening it.
* Uninstalling Citer and reinstalling it.
* Removing all items except one from the .bib file, in case one of them was malformed.

None of which fixed the problem. I have no idea what did.  Sorry I can't reproduce the issue: but I swear it was a real issue.

For what it's worth, I'm now using Citer a set of 600 citations from Zotero, generated using [AutoZotBib](http://www.rtwilson.com/academic/autozotbib).  And Citer is working great.

Thanks for creating Citer, @mangecoeur. I hope someone is able to help you track down this strange bug.
